en/docs/observability/analyze-performance.md

```diff
@@ -16,7 +16,7 @@ The Performance Analyzer tool generates forecasts based on the following assumpt
 
 - For service-based applications the overhead of non-I/O operations is negligible.
 - The service being evaluated is allocated sufficient resources, and therefore, it is not a bottleneck.
-- Sufficient historical data is available for each service operation or API call considered during the analysis.
+- Sufficient historical data is available for each service operation or connector call considered during the analysis.
```

Comment:
What is the reason for changing this? Asking as Console is referring to them as API calls right? 
![image](https://user-images.githubusercontent.com/16300038/124082588-80f72080-da6a-11eb-9482-280d3716033d.png)

Reply:
Nadun mentioned that although the connector calls we have at the moment are all API calls, very soon connector calls which are not API calls (such as MySQL) will be supported, and therefore, it is better to use the term "connector calls".

Reply:
Then why we don't do it along with that future work, with UI text changes also

Reply:
Changed "connector call" to "API call".